post_introspect.yml

```diff
@@ -53,8 +60,83 @@
         msg: "{{ storage_node_disks|unique }}"
       when: ceph_enabled
 
-    - name: remove the templates dir
-      file:
-        state: absent
-        path: /home/stack/templates
-      ignore_errors: true
+    - name: setting node capabilities (boot_mode bios)
+      vars:
+        boot_mode: bios
+      include_tasks: tasks/set_boot_mode.yml
+      when: osp_release|int >= 17 or composable_roles
+
+    - name: get overcloud node images
+      vars:
+        image_path: /home/stack/images
+      include_tasks: tasks/overcloud_images.yaml
+      when: osp_release|int == 17
```

Comment:
Can you change this also to  `when: osp_release|int >= 17`?

Reply:
As we are facing the issue with overcloud images for only osp 17, the condition i have added make sense. Thoughts?

Reply:
Yeah, makes sense.